.github/workflows/mypy.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Install MyPy
         run: |
           pip install mypy
-          pip install ./sdk_commons ./template ./ribbon ./thetanuts
+          pip install . ./template
```

Comment:
Not in love with the "pip install template".
We can think about having a better way after merging 🤗

Reply:
ok